internal/rbd/rbd_util.go

```diff
@@ -554,7 +554,7 @@ func (rv *rbdVolume) getCloneDepth(ctx context.Context) (uint, error) {
 			if errors.Is(err, ErrImageNotFound) {
 				return depth, nil
 			}
-			util.ErrorLog(ctx, "failed to check depth on image %s: %s", vol.String(), err)
```

Comment:
This can be inferred from line 532 `vol := rbdVolume{}`.
Other places, function definition provide the required evidence. :sweat_smile: